src/uct/api/uct_def.h

```diff
@@ -422,13 +585,14 @@ typedef void
  * Other than communication progress routines, it is allowed to call other UCT
  * communication routines from this callback.
  *
- * @param [in]  ep               Transport endpoint.
- * @param [in]  arg              User argument for this callback as defined in
- *                               @ref uct_ep_params_t::user_data
- * @param [in]  status           Indicates the client's status.
+ * @param [in]  ep                 Transport endpoint.
+ * @param [in]  arg                User argument for this callback as defined in
+ *                                 @ref uct_ep_params_t::user_data
+ * @param [in]  connect_cb_handle  Server's connect callback arguments.
  */
 typedef void (*uct_ep_server_connect_cb_t)(uct_ep_h ep, void *arg,
-                                           ucs_status_t status);
+                                           uct_cm_ep_server_connect_cb_handle_t
```

Comment:
const

Reply:
also rename the callback to: uct_cm_ep_server_established_cb_t

Reply:
'established' without the word 'connection' sounds a bit off, no..?
also, since this callback can be called in case of an error, there may be no connection established at all here.

Reply:
we need to find one-word name for every stage of the protocol and use it in a consistent way. we already have "uct_cm_ep" in the name so the context is clear.
for example:
1. server_conn_request
2. client_connect
3. server_establish
BTW I prefer that the connection handlers would have clear name which indicates the stage they are called in the good flow, even if in case of an error the event "didn't really happen"